source/_patterns/03-uswds/header/secondary/secondary.twig

```diff
@@ -0,0 +1,5 @@
+<div class="usa-nav__secondary">
+  {% block content %}
+    {% include "@uswds/header/secondary/_secondary-nav.twig" %}
```

Comment:
What’s the rationale for breaking this out into a separate file?

Reply:
Good question! I think that's actually a holdover from some previous iteration that no longer makes sense. Originally, the rational was so that they could be extended/embedded into different Drupal templates. But since the Drupal menus are all using the menu component and the region template overrides the content block anyway, there's really no reason to pull the menu ul out separately. I'll update.